web-ui/src/pages/PulsePage.jsx

```diff
@@ -97,22 +97,18 @@ const PulsePage = () => {
       internalScore: internalScore + 1, // converts to 1-based
       submissionDate: today,
       updatedDate: today,
-      teamMemberId: myId
+      teamMemberId: submitAnonymously ? null : myId,
     };
-    const res = await resolve({
-      method: 'POST',
-      url: '/services/pulse-responses',
-      headers: {
-        'X-CSRF-Header': csrf,
-        Accept: 'application/json',
-        'Content-Type': 'application/json;charset=UTF-8'
-      },
-      data
-    });
+    const res = await initiate(pulseURL, csrf, data);
     if (res.error) return;
 
-    // Refresh browser to show that pulses where already submitted today.
-    history.go(0);
+    if (submitAnonymously) {
+      setSubmittedToday(true);
+      Cookies.set(cookieName, 'true', { expires: 1 });
+    } else {
+      // Refresh browser to show that pulses where already submitted today.
+      history.go(0);
```

Comment:
I know this wasn't you, but this is odd to me. Why are we refreshing the whole page to trigger a state change, I wonder... I blame @mvolkmann. I think he forgot how to write React code. 🤣

Reply:
Now that I think about this. Couldn't this whole if/else be turned into just lines 106-107?

Reply:
Yes.  I think we can.  Currently, the cookie is only used when anonymous.  But, we could do it generically too.  Also, the refresh was weird to me too.

Reply:
`history.go(0)` is the tool for when you can't find another way to get it to work and you are getting tired. ;-)

Reply:
🤣🤣🤣